queries.md

```diff
@@ -0,0 +1,51 @@
+# Query documentation
+
+## Highlights
+
+### Description of query
+
+The highlights query is composed of two parts:
+
+( A ) Select **1 item** from History where:
+
+* the item has a bookmark
+* the date the bookmark was created (`bookmarkDateCreated`) is in the last 3 days
+* the bookmark has been visited no more than 3 times (`visit_count`)
+
+Items are ordered (descending) by the date the bookmark was created (`bookmarkDateCreated`), meaning that newer item will come first.
+
+( B ) Then, select up to **20 items** from History, where:
+
+* the item was NOT visited (`last_visit_date`) in the last 30 minutes
+* the `rev_host` is unique (no two results returned should have the same `rev_host`
+* the `rev_host` is NOT one of "www.google.com", "www.google.ca", "calendar.google.com", "mail.google.com", "mail.yahoo.com", "search.yahoo.com", "localhost", "t.co", "."
+
+Item are ordered (descending) by last visit date (`last_visit_date`), meaning that more recent items will come first.
+
+### Examples of items that could satisfy the requirements
+
+* A link with a bookmark created 10 seconds ago that was only visited once
+* A link with a bookmark created 2 days ago that was visited three times
+* `www.wired.com`, visited 45 minutes ago
+* `www.cnn.com/some-article` visited 2 hours ago
+
+### Examples of items that would *not* satisfy the requirements
+
+* A link with a bookmark created 5 minutes ago that was visited 4 times *(reason: it was visited more than 3 times)*
+* A link with a bookmark created last month that was visited once *(reason: it is too old)*
```

Comment:
I think that link would satisfy the requirements of the B) part of the highlights query.

* it was last visited more than 3 days ago
* the visit count is <= 3